arm/Microsoft.Automation/automationAccounts/deploy.bicep

```diff
@@ -8,27 +8,19 @@ param location string = resourceGroup().location
   'Free'
   'Basic'
 ])
-@description('Optional. SKU name of the account')
+@description('Optional. SKU name of the account.')
```

Comment:
Add empty line for consistency

Reply:
Not sure where. Do you mean between `@allowed` and `@description`?

Reply:
Nevermind, the comparison view confused me